css/CalendarMonth.scss

```diff
@@ -36,6 +36,15 @@
   margin-bottom: 2px;
 }
 
+.CalendarMonth__weeknumber {
+  padding: 0;
+  box-sizing: border-box;
+  color: $react-dates-color-gray;
+  cursor: default;
+  width: 39px;
```

Comment:
Maybe we should pull these values out into a variable so we can share between the `.CalendarMonth__day` class and this one